src/cosmosdb-preview/azext_cosmosdb_preview/_help.py

```diff
@@ -7,6 +7,154 @@
 
 from knack.help_files import helps  # pylint: disable=unused-import
 
+helps['cosmosdb mongocluster firewall rule'] = """
+type: group
+short-summary: CosmosDB for MongoDB cluster firewall rule.
+"""
+
+helps['cosmosdb mongocluster firewall rule create'] = """
+type: command
+short-summary: Create a CosmosDB for MongoDB cluster firewall rule.
```

Comment:
Create a Mongo cluster firewall rule

Reply:
Same below

Reply:
In general we should be consistent with the friendly names in descriptions, but using 'CosmosDB for MongoDB' is too verbose and doesn't add much at this level.

Reply:
fixed